src/plone/restapi/tests/test_batching.py

```diff
@@ -503,7 +506,8 @@ def test_prev_link_contained_if_necessary(self):
         self.request.form["b_size"] = 10
         self.request.form["b_start"] = 20
         batch = HypermediaBatch(self.request, items)
-        self.assertDictContainsSubset({"prev": "http://nohost?b_start=10"}, batch.links)
+        expected = {"prev": "http://nohost?b_start=10"}
+        self.assertEqual(batch.links, {**batch.links, **expected})
```

Comment:
When testing for a single dict key like this, it would be more readable to just write this as self.assertEqual(batch.links["prev"], "http://nohost?b_start=10")

Reply:
Yes please, because it took me a few minutes to understand what is happening here. :-)

Reply:
Oh, indeed, I tried to keep it as close as possible to the existing behavior, but indeed 👍🏾